### ISSUE TITLE
Wrong example

### ISSUE THREAD

**Issue by GuentherK** [dotnet/EntityFramework.Docs#2800]
[Enter feedback here]

In https://docs.microsoft.com/en-us/ef/core/what-is-new/ef-core-5.0/whatsnew#many-to-many I guess you mean Posts and Tags instead of Posts and Blogs

So this

```
public class BlogContext : DbContext
{
    public DbSet<Post> Posts { get; set; }
    public DbSet<Blog> Blogs { get; set; }
}
```

should be this

```
public class BlogContext : DbContext
{
    public DbSet<Post> Posts { get; set; }
    public DbSet<Tag> Tags{ get; set; }
}
```

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 32360b50-1129-b3d2-2d71-c6b89cd9f2c6
* Version Independent ID: 4f18f8dc-5cd5-cb37-0f59-fb1767609b2b
* Content: [What's New in EF Core 5.0](https://docs.microsoft.com/en-us/ef/core/what-is-new/ef-core-5.0/whatsnew)
* Content Source: [entity-framework/core/what-is-new/ef-core-5.0/whatsnew.md](https://github.com/dotnet/EntityFramework.Docs/blob/master/entity-framework/core/what-is-new/ef-core-5.0/whatsnew.md)
* Product: **entity-framework**
* Technology: **entity-framework-core**
* GitHub Login: @ajcvickers
* Microsoft Alias: **avickers**

**Comment by GuentherK** [comment]
Also in the example SQL that EF will create I hope that EF will use the correct pluralization of table names 

CREATE TABLE [Tag**s**] (
    [Id] int NOT NULL IDENTITY,
    [Text] nvarchar(max) NULL,
    CONSTRAINT [PK_Tag**s**] PRIMARY KEY ([Id])
);

CREATE TABLE [PostTag] (
    [PostsId] int NOT NULL,
    [TagsId] int NOT NULL,
    CONSTRAINT [PK_PostTag] PRIMARY KEY ([PostsId], [TagsId]),
    CONSTRAINT [FK_PostTag_Posts_PostsId] FOREIGN KEY ([PostsId]) REFERENCES [Posts] ([Id]) ON DELETE CASCADE,
    CONSTRAINT [FK_PostTag_Tag**s**_TagsId] FOREIGN KEY ([TagsId]) REFERENCES [Tag**s**] ([Id]) ON DELETE CASCADE
);

CREATE INDEX [IX_PostTag_TagsId] ON [PostTag] ([TagsId]);

**Comment by ajcvickers** [comment]
Fixed by #2798
